sympy/core/tests/test_wester.py

```diff
@@ -2341,11 +2339,8 @@ def test_V11():
             log(((tan(x/2) + 1)/(tan(x/2) + 7))**Rational(1, 3)))
 
 
-@XFAIL
 def test_V12():
     r1 = integrate(1/(5 + 3*cos(x) + 4*sin(x)), x)
-    # Correct result in python2.7.4 wrong result in python3.3.1
-    # https://github.com/sympy/sympy/issues/7157
```

Comment:
How about leaving in a reference to #7157 so if this regresses, people know where to look for more information?